.github/workflows/release.yml

```diff
@@ -0,0 +1,77 @@
+name: release
+
+on:
+    workflow_dispatch:
+        inputs:
+            RELEASE_VERSION:
+                description: 'Version to release'
+                required: true
+
+jobs:
+    release:
+        name: Release ${{ github.event.inputs.RELEASE_VERSION }}
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout repo
+              uses: actions/checkout@v4
+            
+            - name: Install uv
+              uses: astral-sh/setup-uv@v5
+
+            - name: Install dependencies
+              run: |
+                uv venv --python=3.11
+                uv pip install -r requirements/ci.txt -r pipeline/requirements.txt
+
+            - name: Download raw data
+              run: |
+                uv run make download-raw
+
+            - name: Process raw data into duckdb file
+              run: |
+                uv run make process
+
+            - name: Set environment variable for version
+              run: |
+                echo "VERSION_UNDERSCORE=${{ github.event.inputs.RELEASE_VERSION }}" | sed 's/\./_/g' >> $GITHUB_ENV
+            
+            - name: Make sure file exists
+              run: |
+                if [ ! -f data/processed/scipeds_${{ env.VERSION_UNDERSCORE }}.duckdb ]; then
+                  echo "File does not exist!"
+                  exit 1
+                fi
+                ls -l data/processed/
+             
+            - name: Authenticate with GCP
+              uses: google-github-actions/auth@v2
+              with:
+                credentials_json: ${{ secrets.GCP_SA_KEY }}
+            
+            - name: Upload processed data to GCS
+              uses: google-github-actions/upload-cloud-storage@v2
+              with:
+                path: 'data/processed/scipeds_${{ env.VERSION_UNDERSCORE }}.duckdb'
```

Comment:
Just noting that I think it **is correct** that you need to do the `${{ ... }}` here because this is a GitHub Actions thing and not a bash thing.